support-redemptiondb/cfn.yaml

```diff
@@ -0,0 +1,58 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Parameters:
+  Stage:
+    Type: String
+    Description: Stage for the tables
+    AllowedValues:
+      - CODE
+      - PROD
```

Comment:
I decided in the end to have CODE and PROD (like the promo code tool) and then within PROD we have the touchpoint stages of DEV/UAT/PROD

Reply:
I think this is the best approach 👍